renovate.json

```diff
@@ -52,7 +52,7 @@
        "datasourceTemplate": "github-releases"
      },
      {
-       "fileMatch": ["^01_prepare_hosts.sh$"],
+       "fileMatch": ["^lib/common.sh$"],
        "matchStrings": ["ANSIBLE_VERSION:-\"(?<currentValue>.*?)\"}"],
```

Comment:
Looks like you forgot to move `ANSIBLE_VERSION` back to lib/common.sh, or leave this as is

Reply:
Thanks, good find! There was a conflict when reverting and I must have missed this when resolving it.